<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>results</class>
 <widget class="QMainWindow" name="results">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timepad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>380</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="table_b">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Добавить таблицу в базу данных</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stat_btn">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести статистику</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="window_button">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Вывести инфографику</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Segoe Print&quot;;</string>
    </property>
    <property name="text">
     <string>Timepad</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>463</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Добавить предмет</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Добавить ученика</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Вывести таблицу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
